### DISCUSSION TITLE
<Salvar projeto e publicar no GitHub>

### DISCUSSION THREAD

**Discussion by PabloPerdigao** [community/community#67717] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Bom dia!
Algúem sabe me dizer pq não tá dando pra salvar o projeto Devlinks lá do discover que eu fiz? tô tentando colocar ele como público.
Bom dia!
![image](https://github.com/community/community/assets/140118912/b4e3bfad-bd3d-47f1-8920-2bcb5135614f)

**Comment by RumeysaaKopuz** [comment]
response #1
